client/src/js/modules/visits/views/visit_list.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+
+    <div>
+
+        <p class="tw-m-4 tw-p-4 tw-text-lg md:tw-text-2xl 
+                tw-mx-auto tw-border-b tw-border-gray-500">
+                Visit List</p>
+
+        <div class="tw-mt-12 tw-mb-4">
+            
+        </div>
+
+        <div class="tw-overflow-x-scroll tw-scrolling-touch">
+
+            <div class="tw-pb-2">
+                <input placeholder='Search'                         
+                       v-on:keyup.enter="fetchData" 
+                       v-model = "searchVisit"/>
+            </div>
+
+            <table class="tw-w-full tw-mb-2">
+                <thead>
+                    <td v-for="(value) in headers" :key="value.id"
+                    class="tw-w-1/8 tw-bg-table-header-background tw-text-table-header-color tw-font-bold tw-py-2 tw-text-center">
+                    {{value.title}}
+                    </td>
+                </thead>
+
+                <tbody v-for="(visit, rowIndex) in visits" :key="visit.id" >
+                    <tr class="hover:tw-bg-table-row-hover-background"
+                        @click="navigator($event, visit)"
+                        @focusout="handleFocusOut(visit)"
+                    :class="{
+                                'tw-bg-table-body-background-odd' : rowIndex % 2,
+                                'tw-bg-content-sub-header-background' : isOdd % 2,
+                            }"> 
+                        <td v-for="(value) in headers" :key="value.id"
+                            :id="value.key" class="tw-p-1 tw-text-center">
+                                <p v-if="value.key != 'COMMENTS' && value.key != 'ARCHIVED'">{{visit[value.key]}}</p>
```

Comment:
Always use [strict equality (===)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality), even when it doesn't seem like much of a difference, since Javascript does funny things with type conversions and we could be introducing bugs with non-strict equality checks